plugin/references.py

```diff
@@ -55,7 +55,8 @@ def run(
         side_by_side: bool = False,
         force_group: bool = True,
         fallback: bool = False,
-        group: int = -1
+        group: int = -1,
+        include_declaration: bool = False
```

Comment:
Needs to be added in `is_visible` and `is_enabled` above also.

Reply:
Thank you. I have updated the PR.

What is the practice here, shall I resolve the conversation, or shall a reviewer do it?

Reply:
Think best to leave for the issue maker to verify and resolve.